### ISSUE TITLE
Playback does not work on any android-x86 based OS

### ISSUE THREAD

**Issue by Maxcr1** [yuliskov/SmartTubeNext#1823]
SmartTubeNext is fantastic, and you all have done fantastic work. If android-x86 support isn't intended, feel free to disregard this issue. In case you do wish to resolve it, here is everything I have discovered.

I have been able to replicate the issue across multiple builds of android-x86 (_Android 7.1.2_ and _Android 9.0_), across multiple flavors (_AOSP_, _AOSP TV_, _LineageOS x86_, and _LineageOS x86 TV_), and in multiple environments (_[HP EliteDesk 800 G1 SFF](https://support.hp.com/us-en/product/hp-elitedesk-800-g1-small-form-factor-pc/5387475/document/c03832938), VirtualBox 6.1.36 with [these settings](https://user-images.githubusercontent.com/16179691/186996845-edbe7f2b-9309-49f5-8bef-52bb2a0dd5f6.jpg)_)

The problem is, playing videos doesn't work. You can select them, and the playback screen will appear, but will remain black. Below are screenshots from VirtualBox screenshot of the home screen and an attempt to play a video (I turned on the developer options)
![image](https://user-images.githubusercontent.com/16179691/186999525-e73f618c-31ba-4b25-a66e-dc832fc0f6aa.png)
![image](https://user-images.githubusercontent.com/16179691/186999609-c5b1a3d3-3064-4873-81e1-1d6f9cf982dd.png)
Pressing the play button will change nothing, and it will return to the play icon after a second or so.

I have been able to get SmartTubeNext working on this network, so I am confident it is not a DNS/filtering/network-wide issue. I'm not particularly adept at Android development, so I am unsure on how to gather whatever logs may be useful, but if you point me in that direction, I would be happy to do so. [Here is a link to the ISO I used for Android 9 AOSP](https://www.android-x86.org/releases/releasenote-9-0-r2.html).

**Comment by grim1988** [comment]
Just like you're stated, playback doesn't work in the newer versions.  Playback DOES work, for me if I use an older version like 14.84....   Only if you don't log into your YouTube account.  If you're logged in, it does the same thing the newer versions do.

**Comment by mlopeza** [comment]
Just an extra data point. I'm seeing the same behavior, and I also was able to install version 14.84 and playback, but once I login it stops working. I think it might be related to the quality preset? I noticed that most of the videos where on 320p if I wasn't logged in.

**Comment by thestupidbloke** [comment]
Hello @Quackdoc I can see you mentioned this issue in #2478, does that mean it is fixed, or going to be fixed? Please can you explain what the next steps are to get video playback working on android-x86 OS's. Thank you in advance for your help!

**Comment by thestupidbloke** [comment]
Just something that might be of interest, I tried out SmartTube today on my laptop, Lenovo Thinkpad T495, on the AndroidTVx86 OS (got it here: https://archive.org/download/androidtv-x86/ATV-9-x86%20Tech%20info%20Chrome%2023.06.2022-k4.19.105-64bit.iso) and video playback worked! But when I am trying now on my Lenovo Tiny m92p which I am trying to repurpose as a media centre, I had the same issue where everything installs and loads, but videos won't play, I can't load comments etc.

**Comment by thestupidbloke** [comment]
@MaxDeVos, @mlopeza, @grim1988
Please can you try the x86 build of apk, you can get it from here:
https://www.apkmirror.com/apk/yuliskov/stn-stable-android-tv/variant-%7B%22arches_slug%22%3A%5B%22x86%22%5D%7D/

I can confirm it works for me on android tv x86:
https://archive.org/download/androidtv-x86/ATV-9-x86%20Tech%20info%20Chrome%2023.06.2022-k4.19.105-64bit.iso

**Comment by Mishatopkek** [comment]
Still have the same problem. No video playback on x86 Android

**Comment by Has-sh** [comment]
turing off enable native bridge in settings will make the video playback work

**Comment by YahyaKMoney** [comment]
> turing off enable native bridge in settings will make the video playback work

Where in settings? Can you be specific please?

**Comment by Has-sh** [comment]
> > turing off enable native bridge in settings will make the video playback work
> 
> Where in settings? Can you be specific please?

I tried turning off Native Bridge and then tinkered with the sound settings, selecting HDMI as the output(select whatever is your output even if its already selected like deselect and then select again). I’m not sure why this solved the issue. My initial assumption was that disabling Native Bridge fixed it, but now I believe it was actually related to the sound. Please ignore my earlier comment about Native Bridge, as the playback issues were likely caused by the sound output configuration.

**Comment by YahyaKMoney** [comment]
> > > turing off enable native bridge in settings will make the video playback work
> > 
> > 
> > Where in settings? Can you be specific please?
> 
> I tried turning off Native Bridge and then tinkered with the sound settings, selecting HDMI as the output(select whatever is your output even if its already selected like deselect and then select again). I’m not sure why this solved the issue. My initial assumption was that disabling Native Bridge fixed it, but now I believe it was actually related to the sound. Please ignore my earlier comment about Native Bridge, as the playback issues were likely caused by the sound output configuration.

I just realised I'm trying to get this to work on BlissOS, you're probably on something completely different haha.

**Comment by Has-sh** [comment]
> > > > turing off enable native bridge in settings will make the video playback work
> > > 
> > > 
> > > Where in settings? Can you be specific please?
> > 
> > 
> > I tried turning off Native Bridge and then tinkered with the sound settings, selecting HDMI as the output(select whatever is your output even if its already selected like deselect and then select again). I’m not sure why this solved the issue. My initial assumption was that disabling Native Bridge fixed it, but now I believe it was actually related to the sound. Please ignore my earlier comment about Native Bridge, as the playback issues were likely caused by the sound output configuration.
> 
> I just realised I'm trying to get this to work on BlissOS, you're probably on something completely different haha.

yeah it might be but do try to tincker with your sound settings it mind do the job
